.github/workflows/linux.yml

```diff
@@ -0,0 +1,24 @@
+name: linux
+on:
+  - push
+  - pull_request
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '2.4', '2.5', '2.6', '2.7' ]
```

Comment:
What about dropping 2.4 and adding 3.0?

Reply:
Fixed in https://github.com/fluent/fluent-plugin-sd-dns/pull/2/commits/5e17e99293a580e4811baa4f8084cf8b75660c98.